frontend/src/reviews/ReviewRoleDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import NotificationsNoneOutlinedIcon from '@mui/icons-material/NotificationsNoneOutlined'
+import { Stack, Typography } from '@mui/material'
+
+import { useGetModelRoles } from '../../actions/model'
+import { ReviewRequestInterface } from '../../types/interfaces'
+import { getRoleDisplay } from '../../utils/beta/roles'
+import Loading from '../common/Loading'
+import MessageAlert from '../MessageAlert'
+
+type ReviewRoleDisplayProps = {
+  review: ReviewRequestInterface
+}
+
+export default function ReviewRoleDisplay({ review }: ReviewRoleDisplayProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(review.model.id)
+
+  if (isModelRolesError) {
+    return <MessageAlert message={isModelRolesError.info.message} severity='error' />
+  }
+
+  if (review.responses.length > 0) {
+    return <></>
+  }
+
+  return (
+    <>
+      {isModelRolesLoading && <Loading />}
+      <Stack direction='row' alignItems='center' justifyContent='center'>
```

Comment:
You could use the `Stack`'s `spacing` prop to ensure all items within the stack are spaced evenly, instead of applying margin to individual items:
```
      <Stack direction='row' alignItems='center' justifyContent='center' spacing={1}>
        <NotificationsNoneOutlinedIcon sx={{ fontSize: 'medium' }} color='warning' />
```